Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/RichText/RichTextFormatter.cs

```diff
@@ -0,0 +1,170 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats.RichText
+{
+    using Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarButtons;
+    using Windows.System;
+    using Windows.UI.Text;
+    using Windows.UI.Xaml.Controls;
+
+    // Rudimentary showcase of RichText and Toggleable Toolbar Buttons.
+    public class RichTextFormatter : Formatter
+    {
+        public RichTextFormatter(TextToolbar model)
+            : base(model)
+        {
+            ButtonActions = new RichTextButtonActions(this);
+        }
+
+        public void FormatUnderline(ToolbarButton button)
```

Comment:
This is because Underline is not a Common Button, I have moved it to ButtonActions, it just complicates how to assign that method to the Button.